### PR TITLE
Unique page titles

### PR DESCRIPTION
**Pull request by guy-roberts**
### Context
[ I made a mess of the previous PR, so closed that one and opened this, these changes are already in the production branch, via a hotfix ]

Make sure that each page has a unique title

Fix the canonical meta tag to use the correct URL

### CODE REVIEW

File: app/views/layouts/cms.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en" class="govuk-template ">
   <head>
-    <title>Help for early years providers (editing content)</title>
+    <title>Editing Content | Help for early years providers | GOV.UK</title>
```
Comment (dmallett1) [inline app/views/layouts/cms.html.erb:4]: https://www.gov.uk/service-manual/design/writing-for-user-interfaces#headings-and-title provides example, 'Where do you live? - register to vote - GOV.UK'.  Note: The use of hyphens and not pipes, to separate h1 from service name, and to separate service name from 'GOV.UK'

File: app/views/layouts/content.html.erb [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <html lang="en" class="govuk-template">
 <head>
   <meta name="Cache-Control" content="max-age=3500, public">
-  <title>Help for early years providers</title>
+  <title><%= @page.title %> | Help for early years providers | GOV.UK</title>
```
Comment (dmallett1) [inline app/views/layouts/content.html.erb:5]: Same as above.

File: app/views/layouts/landing_page_layout.html.erb [hunk 3]
```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="utf-8">
   <title>
-    Help for early years providers
+    Help for early years providers - UK.GOV
```
Comment (dmallett1) [inline app/views/layouts/landing_page_layout.html.erb:6]: 1. Technically to match the service manual standard this should be: Help for early years providers - Help for early years providers - GOV.UK
2. Note GOV.UK and not typo of UK.GOV

File: app/views/layouts/settings.html.erb [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en" class="govuk-template ">
 <head>
-  <title>Help for early years providers</title>
+  <title>Help for early years providers - UK.GOV </title>
```
Comment (dmallett1) [inline app/views/layouts/settings.html.erb:4]: As above.

File: app/views/layouts/landing_page_layout.html.erb (hunk 3 above)
Comment (guy-roberts) [reply app/views/layouts/landing_page_layout.html.erb:6]: I'll change GOV.UK

But repeating Help for early years providers - Help for early years providers 

is not a good idea.  Its for Google Analytics and saying the same thing twice is not helpful

**Review by dmallett1 — COMMENTED** [review]
The application.html.erb head title should also reflect the service manual standard, for the static pages.  It is quite possible that an accessibility user will search for accessibility and the application.

File: app/views/layouts/landing_page_layout.html.erb (hunk 3 above)
Comment (dmallett1) [reply app/views/layouts/landing_page_layout.html.erb:6]: Within jira story the attachment shows 'Department for Education' after 'Help for early years providers' within landing_page_layout.html

Comment (guy-roberts) [reply app/views/layouts/landing_page_layout.html.erb:6]: Text in page titles should not be repeated

It does not make sense to Google Analytics

Comment (guy-roberts) [reply app/views/layouts/landing_page_layout.html.erb:6]: The Jira story is wrong

File: app/views/layouts/settings.html.erb (hunk 4 above)
Comment (guy-roberts) [reply app/views/layouts/settings.html.erb]: Done

File: app/views/layouts/landing_page_layout.html.erb [hunk 5]
```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="utf-8">
   <title>
-    Help for early years providers
+    Help for early years providers - GOV.UK
   </title>
```
Comment (guy-roberts) [inline app/views/layouts/landing_page_layout.html.erb:7]: This is why it is not a good idea to repeat phrases in the page title

Screenshot 2021-05-18 at 18 24 48

**Review by ebrett — APPROVED** [review]

**Pull request merged**